BlobInfo blob
geometry_msgs/Point32 position
bool have_pos
